src/kubernetes.cc

```diff
@@ -463,7 +476,11 @@ KubernetesReader::GetPodAndContainerMetadata(
         LOG(INFO) << "Container status: " << *container_status;
       }
     }
-    result.emplace_back(GetLegacyResource(pod, name));
+    try {
+      result.emplace_back(GetLegacyResource(pod, name));
+    } catch (const QueryException& e) {
+      LOG(INFO) << e.what();
```

Comment:
Document why it's ok to ignore this exception.

Reply:
Done.